controllers/alertrulegroup_controller.go

```diff
@@ -151,7 +158,7 @@ func (r *GrafanaAlertRuleGroupReconciler) Reconcile(ctx context.Context, req ctr
 	return ctrl.Result{RequeueAfter: r.Cfg.requeueAfter(group.Spec.ResyncPeriod)}, nil
 }
 
-func crToModel(cr *grafanav1beta1.GrafanaAlertRuleGroup, folderUID string) models.AlertRuleGroup {
+func crToModel(cr *grafanav1beta1.GrafanaAlertRuleGroup, folderUID string, log logr.Logger) models.AlertRuleGroup {
```

Comment:
The log looks temporary, but in case you want to keep it, we usually pass down loggers via `context.Context`
And extract it using `log := logf.FromContext(ctx)`
```suggestion
func crToModel(ctx context.Context, cr *grafanav1beta1.GrafanaAlertRuleGroup, folderUID string) models.AlertRuleGroup {
```